src/styles/index.scss

```diff
@@ -1,3 +1,140 @@
 body {
   margin: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  flex-shrink: 0;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+  margin: auto;
+
+  &__foto {
+    display: block;
+    box-sizing: border-box;
+    margin: 32px auto 40px;
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    background-position: center;
+  }
+
+  &__text {
+    color: #060b35;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 16px 4px;
+  }
+
+  &__code {
+    color: #616070;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
```

Comment:
normal is a default value for the font-style, so it's redundant 
400 is a default value for the font-weight, so it's redundant 
check other places